src/external/monaco.flux.messages.ts

```diff
@@ -159,29 +158,17 @@ export const signatureHelp = (
   })
 }
 
-export const parseResponse = (response: ServerResponse): LSPResponse => {
-  const message = response.get_message()
-  const error = response.get_error()
-
-  if (error) {
-    throw new Error(error)
-  }
-
-  const split = (message || '').split('\r\n')
-
-  try {
-    return JSON.parse(split.slice(2).join('\n'))
-  } catch (e) {
-    throw new Error('failed to parse LSP response')
+export const parseResponse = (response: string): LSPResponse => {
+  if (response) {
```

Comment:
the new library either returns a message or throws an error - no more "message.get_error()"

Reply:
Question, where does it land with table columns named error that aren’t necessarily errors?